src/lib/shadow-shim-helper-rs/src/shim_shmem.rs

```diff
@@ -79,6 +79,9 @@ pub struct HostShmem {
     // per-process option.
     pub unblocked_vdso_latency: SimulationTime,
 
+    // Native pid of the Shadow process.
+    pub shadow_pid: libc::pid_t,
```

Comment:
How about "the Shadow simulator process"? When I first read "Shadow process", I thought it meant "Shadow `Process`" and was confused.

Reply:
Done